packages/renderer/src/App.svelte

```diff
@@ -145,11 +146,11 @@ window.events?.receive('display-troubleshooting', () => {
         <Route path="/compose/:name/:engineId/*" breadcrumb="Compose Details" let:meta navigationHint="details">
           <ComposeDetails composeName="{decodeURI(meta.params.name)}" engineId="{decodeURI(meta.params.engineId)}" />
         </Route>
-        <Route path="/pods/:kind/:name/:engineId/*" breadcrumb="Pod Details" let:meta navigationHint="details">
+        <Route path="{PodDetailsRoute.path()}" breadcrumb="Pod Details" let:meta navigationHint="details">
           <PodDetails
-            podName="{decodeURI(meta.params.name)}"
-            engineId="{decodeURI(meta.params.engineId)}"
-            kind="{decodeURI(meta.params.kind)}" />
+            podName="{PodDetailsRoute.getName(meta.params)}"
+            engineId="{PodDetailsRoute.getEngineId(meta.params)}"
+            kind="{PodDetailsRoute.getKind(meta.params)}" />
```

Comment:
Maybe it will be easier to have a getRouteDetails on utility class and then just pass the argument

like

```svelte
<Route path="podUtils.path" breadcrumb="Pod Details" let:meta navigationHint="details">
          {@const routeDetails = podUtils.getRouteDetails(meta.params)}
          <PodDetails
               podName="{routeDetails.podName}"
               engineId="{routeDetails.engineId}"
```